cypress/e2e/addFuelExpence.cy.js

```diff
@@ -0,0 +1,15 @@
+import './commands'
+import addFuelExpencesSteps from '../support/steps/addCarFuelSteps';
+import addCarSteps from '../support/steps/addCarSteps';
+
+describe("Login and add car", () => {
+		beforeEach(() => {
+		cy.login(Cypress.env('email'), Cypress.env('password'));
+		addCarSteps.addCarToGarage()
```

Comment:
it id better to pass data to method `addCarToGarage`

in that case you would be able to provide any data